bb-core/src/main/java/com/cognifide/qa/bb/loadable/hierarchy/WebElementInterceptor.java

```diff
@@ -92,4 +99,21 @@ private ClassFieldContext acquireDirectContext(BobcatWebElement caller) {
     }
     return directContext;
   }
+
+  private boolean isApplicable(Class clazz) {
+    if (clazz.isAnnotationPresent(PageObject.class)) {
```

Comment:
when method's caller class is a page object we need to look up if one of the caller classes above in the stack applies for loadable chain execution